data-plane/integrations/mcp/mcp-proxy/Cargo.toml

```
@@ -19,4 +20,5 @@ rmcp =  { version = "0.1.5", features = [
 ] }
 serde_json = "1.0"
 tokio = { version = "1", features = ["full"] }
+tonic = "0.13.0"
```

If the only reason to introduce tonic is the async trait, we can use them directly from the [async_trait](https://docs.rs/async-trait/latest/async_trait/) crate:

```suggestion
async_trait = "0.1"
```